doc/Running_Motr_Across_a_Cluster.md

```diff
@@ -0,0 +1,273 @@
+This wiki provides information on how to build motr from source and then run a cluster of motr nodes.
+
+## Prerequisites
+
+1.  Create a fresh VM with CentOS 7.9.2009. ( Note: These steps were tested on Windows Hyper-V and VMware Workstation Pro 16, but should in theory work on any hypervisor)
+
+2.  Follow the steps outline in this [guide](https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst) to build Motr from sources on your node.
+
+## Procedure
+
+1.  ### Install HARE
+    * Choose any node in your cluster, we will call this node the Main node for ease of naming. Follow the [HARE installation guide](https://github.com/Seagate/cortx-hare/blob/main/README.md), and install HARE on all nodes. Once you get to the [Quick Start](https://github.com/Seagate/cortx-hare/blob/main/README.md#quick-start) you can return to this guide and continue with the next step below.
```

Comment:
@bdekvadiya , no need to install HARE on all nodes. As we are cloning in the next step, please remove "on all nodes."